### ISSUE TITLE
Can't Include --target="iPad Air 2" (or similar)

### ISSUE THREAD

**Issue by enhendrickson** [ngx-rocket/scripts#6]
Unless I don't understand how to do it (a possibility), I can't quite figure out how, if I can at all, include the --target="<emulator target>" in the cordova command.  This would allow me to emulate, for example, an iPad rather than always the iPhone X which is the default.  

Am I just missing how to do this?

**Comment by enhendrickson** [comment]
Some explanation / examples of what I'm looking for are here: https://stackoverflow.com/questions/22310526/cordova-start-specific-ios-emulator-image

**Comment by sinedied** [comment]
You should be able to pass any option accepted by the cordova CLI after a `--` flag.
Example:
```
ngx-scripts cordova run ios -- --target="ipad-air-2"
```

**Comment by enhendrickson** [comment]
Ahh, I was missing the dashed name.  For some reason, I never saw the device name listed online using the dashes.  Thanks.  This is working now!

**Comment by enhendrickson** [comment]
Also, if you need to run it from the npm side based on the stock package.json, you have to include an extra set of -- double-dashes. E.g., 

npm run cordova:run ios -- -- --target="iPad-Air-2"

**Comment by sinedied** [comment]
Yes, that's why you can use `ngx` instead of `npm run`, to remove one extra `--` 😉 :
```
ngx cordova:run ios -- --target="iPad-Air-2"
```

**Comment by enhendrickson** [comment]
Oh, I had no clue (not in any documentation that I saw) that you can run it from ngx.  Makes sense now that you say it, but cool... thanks!

**Comment by sinedied** [comment]
It's the CLI docs and detailed help: https://github.com/ngx-rocket/generator-ngx-rocket/tree/master/cli#usage

But yeah, it's not easy to find out, we should advertise a bit on this feature on the main page
